easybuild/easyconfigs/n/NAMD/NAMD-2.12-foss-2017a-mpi.eb

```diff
@@ -9,7 +9,11 @@ description = """NAMD is a parallel molecular dynamics code designed for high-pe
 toolchain = {'name': 'foss', 'version': '2017a'}
 toolchainopts = {'usempi': True, 'pic': True}
 
-sources = ['NAMD_%(version)s_Source.tar.gz']
+sources = [{
+    'filename': 'NAMD_%(version)s_Source.tar.gz',
+    'extract_cmd': "tar xfv %s",  # source file is actually not gzipped
```

Comment:
Hmm, seems like I have been modifying the downloaded sources myself (i.e. by zipping it again) up until now, because the source file I have *is* gzipped. It's not when I redownloaded it though.

Does this ring a bell for you @akesandgren?

Reply:
I just subscribed to the NAMD mailinglist and posted a question for this. I'll post the replies I'll get here.

Reply:
The file I have on disk is NAMD_2.12_Source.tar.gz
But i don't remember having done anything to it.
And the older files i have are also tar.gz

Reply:
Yes, the extension is tar.gz, but can you run the file command on it? When I do this, I get

```
$ file NAMD_2.12_Source.tar.gz 
NAMD_2.12_Source.tar.gz: POSIX tar archive (GNU)
```

So not gzipped, despite the extension.

Reply:
@akesandgren Indeed, the downloaded file has `.tar.gz` extension, but it's really just a (non-gzipped) tarball.

I see the same thing when re-downloading it, so I suspect I just fixed that myself manually by gzipping it after downloading.

This approach here is better, but then we should also make it consistent (i.e. apply the same change, including a checksum, in the existing NAMD 2.12 easyconfigs).

Reply:
... which is exactly what @tovrstra did, thanks. :)

Reply:
You can follow up my message on the mailing list here: http://www.ks.uiuc.edu/Research/namd/mailing_list/namd-l.2016-2017/2442.html
No replies yet. Nobody seems to care, so don't expect this to change anytime soon.

I've looked around a bit: you can also get the source code here apparently: https://github.com/divijverma/NAMD/releases I'm not sure how legal and reliable that is. At least you'll get a .tar.gz when you're told so :)

Reply:
We should stick to the official sources, and just deal with the faulty naming, it's an easy/clean fix now in the easyconfig.